107/src/main/java/org/ehcache/jsr107/internal/Jsr107ServiceConfigurationParser.java

```diff
@@ -89,11 +98,42 @@ public URI getNamespace() {
 
   @Override
   public Class<Jsr107Service> getServiceType() {
-    return null;
+    return Jsr107Service.class;
   }
 
   @Override
   public Element unparseServiceCreationConfiguration(ServiceCreationConfiguration<Jsr107Service> serviceCreationConfiguration) {
-    throw new UnsupportedOperationException("Not yet implemented");
+    return unparseConfig(serviceCreationConfiguration);
   }
+
+  @Override
+  public Element createRootElement(Document doc, Jsr107Configuration configuration) {
+    Element rootElement = doc.createElementNS(NAMESPACE.toString(), JSR_NAMESPACE_PREFIX + COLON + DEFAULT_ELEMENT_NAME);
```

Comment:
This might work... but is it supposed to? I'd be much more confident if we just used the `NS` method variants consistently.